src/app/views/sidebar/history/History.tsx

```diff
@@ -406,6 +424,17 @@ export class History extends Component<IHistoryProps, any> {
     }
   }
 
+  private trackHistoryItemEvent = (eventName: string, componentName: string,
+      query: IHistoryItem) => {
+    telemetry.trackEvent(
+      eventName,
+      {
+        ComponentName: componentName,
+        ItemIndex: query.index,
+        QueryUrl: ''
```

Comment:
we should think about a different name for this so it's clear we are not capturing the query url.  more like the query signature.